admin_pages/events/Events_Admin_Page.core.php

```diff
@@ -952,25 +955,28 @@ public function extra_permalink_field_buttons($return, $id, $new_title, $new_slu
      *
      * @access protected
      * @return void
-     * @throws \EE_Error
+     * @throws EE_Error
      */
     protected function _events_overview_list_table()
     {
         do_action('AHEE_log', __FILE__, __FUNCTION__, '');
-        $this->_template_args['after_list_table'] = ! empty($this->_template_args['after_list_table'])
-            ? (array) $this->_template_args['after_list_table']
-            : array();
+        $this->_template_args['after_list_table']                           =
+            ! empty($this->_template_args['after_list_table'])
+                ? (array) $this->_template_args['after_list_table']
+                : [];
         $this->_template_args['after_list_table']['view_event_list_button'] = EEH_HTML::br()
-                . EEH_Template::get_button_or_link(
-                    get_post_type_archive_link('espresso_events'),
-                    esc_html__("View Event Archive Page", "event_espresso"),
-                    'button'
-                );
-        $this->_template_args['after_list_table']['legend'] = $this->_display_legend($this->_event_legend_items());
-        $this->_admin_page_title .= ' ' . $this->get_action_link_or_button(
+            . EEH_Template::get_button_or_link(
+                get_post_type_archive_link('espresso_events'),
+                esc_html__("View Event Archive Page", "event_espresso"),
+                'button'
+            );
+        $this->_template_args['after_list_table']['legend']                 = $this->_display_legend(
+            $this->_event_legend_items()
+        );
+        $this->_admin_page_title                                            .= ' ' . $this->get_action_link_or_button(
```

Comment:
ya I agree.. the PhpStorm auto-formatter used to do a better job before imho
i can experiment with settings to see if there is some combination that works better.

The one solution i do know of is to simply add a empty lines around the block that is getting formatted funny,
because the auto-formatter appears to only apply that spacing to lines within the same "block" of code, and empty lines or comments will separate one block from another.

I try to fix these when I see them but sometimes they slip through

Reply:
Ok, understood.